CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.13.0] - 2022-08-23
```

Comment:
We add the release line in a separate step, typically. First we merge the change to develop, then we make a release branch. We mostly follow the process documented here: https://github.com/Cray-HPE/community/wiki/Gitflow-Development-Process

Except that we don't use Github Actions for the builds yet.

Anyway, if this ends up being needed for 1.3, I can help you get this part done.
```suggestion
```